addons/spreadsheet/static/src/list/plugins/list_ui_plugin.js

```diff
@@ -125,7 +126,9 @@ export class ListUIPlugin extends OdooUIPlugin {
 
                 const domainEditionCommands = cmd.commands.filter(
```

Comment:
This should be renamed
```suggestion
                const updateCommands = cmd.commands.filter(
```